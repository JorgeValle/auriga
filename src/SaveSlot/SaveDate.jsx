import React from 'react';

// Components should represent one piece of data model
class SaveDate extends React.Component {
  render() {

    const time = this.props.time;

    return (
      <div className="save-date">
        January 17
      </div>
    );
  }
}

export default SaveDate;