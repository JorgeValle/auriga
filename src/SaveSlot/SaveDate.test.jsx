import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import SaveDate from './SaveDate.jsx';

configure({
  adapter: new Adapter()
});

describe('<SaveDate />', () => {

  // We shallow render to make sure changes in children don't cause cascading fails
  it('shallow renders', () => {
    const wrapper = shallow(<SaveDate />);
  });

  // Snapshot matching
  it('matches snapshot', () => {
    const wrapper = shallow(<SaveDate />);
    expect(wrapper).toMatchSnapshot();
  });

});

