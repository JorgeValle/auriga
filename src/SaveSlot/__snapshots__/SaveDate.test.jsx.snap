// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SaveDate /> matches snapshot 1`] = `ShallowWrapper {}`;
